packages/swapper/src/api.ts

```diff
@@ -94,6 +104,7 @@ interface TradeBase<C extends ChainId> {
   buyAsset: Asset
   sellAsset: Asset
   sellAssetAccountNumber: number
+  buyAssetAccountNumber: number
```

Comment:
Already a pattern, so I don't think we need to change in this PR, but I don't understand the reason our trade responses return the account numbers. Would make more sense to just either use the default bip44 params at time of quote/execute or have the client determine the account. Just commenting more for visibility

Reply:
I think the idea is the client passes in the account numbers and we generate the bip44 params from those.  @elmutt probably has more info on this

Reply:
@kaladinlight and i have talked generally about this issue (with respect to thorchain) and are on the same page for the most part. 

I dont think we need to change anything here